tests/test_database_manager.py

```diff
@@ -0,0 +1,140 @@
+"""
+test_database_manager.py - tests for the DatabaseManager package.
+
+A database configuration and connection is required for these tests.
+No data will be written to or deleted from the database.
+
+Copyright (c) 2018 The Fuel Rats Mischief,
+All rights reserved.
+
+Licensed under the BSD 3-Clause License.
+
+See LICENSE
+"""
+import pytest
+import psycopg2
+from database import DatabaseManager
+from psycopg2 import extensions, pool, sql
+
+pytestmark = pytest.mark.database_manager
+
+
+@pytest.fixture
+def test_dbm() -> DatabaseManager:
+    """
+    Test fixture for Database Manager.
+
+    A DATABASE CONFIGURATION AND CONNECTION IS REQUIRED FOR THESE TESTS.
+    """
+    database = DatabaseManager()
+    return database
```

Comment:
Shouldn't the fixture also handle cleaning up the DBM instance during teardown?
Should the DBM be function scoped or test-session scoped?
Shouldn't this fixture be in `conftest.py`?

Fixtures should follow established naming convention, namely being suffixed with `_fx` to signify they are actually fixtures.

Reply:
Fixed in 79daed8 and 6da7c08.

No teardown is necessary.  Connections are closed upon destroying the connection pool by the library.